helm/default-apps-cloud-director/values.yaml

```diff
@@ -156,6 +156,21 @@ apps:
     # used by renovate
     # repo: giantswarm/observability-bundle
     version: 0.8.7
+  teleportKubeAgent:
+    appName: teleport-kube-agent
+    chartName: teleport-kube-agent
+    catalog: default
+    clusterValues:
+      configMap: true
+      secret: true
+    forceUpgrade: true
+    namespace: kube-system
+    # used by renovate
+    # repo: giantswarm/teleport-kube-agent-app
+    version: 0.6.0
+    extraConfigs:
+    - kind: configMap
+      name: "{{ $.Values.clusterName }}-teleport-kube-agent-config"
```

Comment:
sorry. just came to my mind. who is supposed to create this cm?

Reply:
If so, we shouldn't merge this PR until teleport-operator is a part of the vsphere-collection https://github.com/giantswarm/vsphere-app-collection/tree/main/flux-manifests 

Right?

Reply:
teleport-operator is now part of vsphere-collection.
- https://github.com/giantswarm/vsphere-app-collection/commit/188e5d283dcebfb9ea77769d69acff4dc7a56860